<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <link rel="stylesheet" href="https://hive.biochemistry.gwu.edu/prd//gcsslib/bootstrap.min.css" type="text/css">
  <link rel="stylesheet" href="https://hive.biochemistry.gwu.edu/prd//gcsslib/global.css" type="text/css">
  <link rel="stylesheet" href="https://hive.biochemistry.gwu.edu/prd//gcsslib/googlefonts.css" type="text/css">
</head>

<body>
<div vocab="http://schema.org/" typeof="WebPage">
<table style="width:100%;font-size:13px;">
<tr>
<td style="width:100%; vertical-align: top;" colspan="2">
<h3 id="Top">HIVE Team Datasets:</h3>
</td>
</tr>
<tr style="height: 78px;">
<td style="width: 40px; text-align: right; vertical-align: top; height: 78px;">
<p>&nbsp;</p>
</td>
<td style="vertical-align: top; width:100%; height: 78px;font-size:13px;">
<ol>

<li>
<div vocab = "http://schema.org/" typeof="Dataset">
  <span property="name">BCO HCV</span><br>
  <span property="abstract">
	We demonstrated that the use of the IEEE 2791-2020 Standard, (BioCompute objects [BCO]) enables complete and concise communication of NGS data analysis results. One arm of a clinical trial4 was replicated using synthetically generated data made to resemble real biological data. Two separate, independent analyses were then carried out using BCOs as the tool for communication of analysis: one to simulate a pharmaceutical regulatory submission to the FDA, and another to simulate the FDA review. The two results were compared and tabulated for concordance analysis: of the 118 simulated patient samples generated, the final results of 117 (99.15%) were in agreement. This high concordance rate demonstrates the ability of a BCO, when a verification kit is included, to effectively capture and clearly communicate NGS analyses within regulatory submissions. BCO promotes transparency and induces reproducibility, thereby reinforcing trust in the regulatory submission process.
  </span><br>
  <br><a href="">Sample Read files</a>
  Version: 
  <span property="version">1.0 </span>
  Size:
  <span property="size">5 GB</span>
  Format: 
  <span property="encodingFormat">FASTQ</span>
  Citation: 
  <span property="citatioon">PMID: ######</span>

  <br><a href="">Sample Read files</a>
  Version: 
  <span property="version">1.0 </span>
  Size:
  <span property="size">5 GB</span>
  Format: 
  <span property="encodingFormat">FASTQ</span>
  Citation: 
  <span property="citatioon">PMID: ######</span>

  <br><a href="">Sample Read files</a>
  Version: 
  <span property="version">1.0 </span>
  Size:
  <span property="size">5 GB</span>
  Format: 
  <span property="encodingFormat">FASTQ</span>
  Citation: 
  <span property="citatioon">PMID: ######</span>

</div>
</li>
<br>
<li>
  <span property="name">GFKB</span><br>
  <span property="abstract">
	Gut feeling knowledgebase is a reference database of healthy human gut microbiome. It is generated by a metagenomic analysis pipeline describes in our paper (doi: 10.1371/journal.pone.0206484), and includes three tools which are integrated in the HIVE platform. 49 healthy samples sequenced at GWU and 49 healthy samples taken from The Human Microbiome Project were analysed to create GutFeelingKB.
  </span><br>
  <br>Version: 
  <span property="version">4.0 </span>
  Citation: 
  <span property="citatioon">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/31509535/">31509535</a></span>
  <br>Downloads Link:
  <a href="https://hive.biochemistry.gwu.edu/gfkb">https://hive.biochemistry.gwu.edu/gfkb</a>
  
</li>

</ol>
</td>
</tr>
<tr>
<td style="width:100%; vertical-align: top;" colspan="2">
<br>
</body>